server/src/main/java/org/candlepin/controller/ManifestManager.java

```diff
@@ -0,0 +1,388 @@
+/**
+ * Copyright (c) 2009 - 2012 Red Hat, Inc.
```

Comment:
> Copyright header should be 2009-2016

Need to update the checkstyle rules in order to do that. It has been that way for a very long time and we've never found time (or made time) to do it.